src/Components/Modals/AddAdditionalRepositoriesModal.js

```diff
@@ -11,6 +11,28 @@ import useAddAdditionalRepositories from '../../hooks/useAddAdditionalRepositori
 import useNotifications from '../../hooks/useNotifications';
 import AddAdditionalRepositoriesTable from '../AddAdditionalRepositoriesTable';
 
+const ProgressButton = ({ selectedRepositories, submitForm, isSubmitting }) => {
+  const [buttonState, setButtonState] = useState('notClicked');
+  const handleClick = () => {
+    setButtonState('clicked');
+    submitForm();
+  };
+
+  return (
+    <ActionGroup>
+      <Button
+        variant="primary"
+        onClick={handleClick}
+        isLoading={buttonState === 'clicked'}
```

Comment:
Shouldn't `isLoading` be the same as `isSubmitting`? I don't think we need the `clicked` vs `notClicked` check. Cause that just means submitting right?